packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -15,10 +17,13 @@ export async function findProjectRootAsync(cwd?: string): Promise<string | null>
   return projectRootDir ?? null;
 }
 
-export async function getProjectIdAsync(projectDir: string): Promise<string> {
+export const getProjectAsync = once(_getProjectAsync);
+
+async function _getProjectAsync(projectDir: string): Promise<AppQueryResult> {
   const { exp } = getConfig(projectDir, { skipSDKVersionRequirement: true });
```

Comment:
It's  not specific to this PR, but I think that all functions from this file that call getCOnfig should have this passed as argument.

Reply:
We can think about cleaning up `getConfig` calls in whole project - now it's called in many places, during single command execution. I made an issue sometime ago: #81 .
But it's definitely not related to this PR.